tests/test_util_name.rs

```diff
@@ -92,3 +92,154 @@ fn util_name_single() {
         scenario.fixtures.plus("uu-sort").display()
     )));
 }
+
+#[test]
+#[cfg(any(unix, windows))]
+fn util_invalid_name_help() {
+    use std::{
+        io::Write,
+        process::{Command, Stdio},
+    };
+
+    let scenario = TestScenario::new("invalid_name");
+    symlink_file(scenario.bin_path, scenario.fixtures.plus("invalid_name")).unwrap();
+    let child = Command::new(scenario.fixtures.plus("invalid_name"))
```

Comment:
Would it make sense to adapt the test framework to support this so that we can write this test witht it?

Reply:
Hmm, sure, it is possible to do so, but I don't see the benefit?

The tests "unrecognized name" and "non-UTF-8 name" are likely going to be the only use cases for this. Perhaps one more edge case that I overlooked, but I really don't see the advantage to make this more generally available.